### PR TITLE
Display robot pose in Frontend

### LINKED ISSUE #304: Present Robot Pose in Frontend
**Describe the new feature you would like to see**
The robot pose should be presented to the user when entering the robot view page.
If only for debug purposes in the beginning.

**Describe the solution you'd like**
Add a box in mission/robot page that list the pose.
Should also edit isar_robot to publish random pose values, so that we can verify changes.

**How will this feature affect the current Threat Model?**
More data will be exposed to the user.

### PR DESCRIPTION
**Pull request by tsundvoll**
Resolves  #304

### CODE REVIEW

File: frontend/src/components/MapPosition/MapPositionSection.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    gap: 1rem;
+`
+
+interface MissionProps {
+    mission?: Mission
+}
+
+export function MapPositionSection({ mission }: MissionProps) {
+    var text
+    if (mission && mission.robot.pose)
```
Comment (oysand) [inline frontend/src/components/MapPosition/MapPositionSection.tsx:17]: Should we wrap this in a 
useEffect(()={
updateText
}, []) 
to ensure that it only runs once? alternately 

useEffect(()={
updateText
}, [changingPoseVariable])

Comment (tsundvoll) [reply frontend/src/components/MapPosition/MapPositionSection.tsx:17]: Good question, let us think more about this.

File: frontend/src/components/MapPosition/MapPositionSection.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    display: grid;
+    grid-column: 1/ -1;
+    gap: 1rem;
+`
+
+interface MissionProps {
+    mission?: Mission
+}
+
+export function MapPositionSection({ mission }: MissionProps) {
```
Comment (oysand) [inline frontend/src/components/MapPosition/MapPositionSection.tsx:15]: Should we add the apiCaller and a new endpoint to get the position? That way each component calls for what it needs and decides how often it wants the information. I see that this also applies to the task table. So if we want this we should also add an issue for it

**Review by eivindsjovold — CHANGES_REQUESTED** [review]

File: frontend/src/components/MapPosition/MapPositionSection.tsx (hunk 1 above)
Comment (tsundvoll) [reply frontend/src/components/MapPosition/MapPositionSection.tsx:17]: As it is written now it works the same way as the tasks update. I would say that is fine for now, and will create an issue to do this through an API call directly as suggested in the next comment.

File: frontend/src/components/MapPosition/MapPositionSection.tsx (hunk 2 above)
Comment (tsundvoll) [reply frontend/src/components/MapPosition/MapPositionSection.tsx:15]: That seems like a good approach. I have created issue #458 on this.

**Comment by tsundvoll** [comment]
I think this approach works for now and we can take it in if you agree. New issue https://github.com/equinor/flotilla/issues/458 on doing this in a better way, also for other components.

**Review by eivindsjovold — APPROVED** [review]
:100:

**Pull request merged**